src/CommonITILObject.php

```diff
@@ -188,12 +188,13 @@ public function getActorsForType(int $actortype = 1, array $params = []): array
                         );
                         $email = UserEmail::getDefaultForUser($users_id_default);
                         $actors[] = [
-                            'items_id'          => $users_id_default,
-                            'itemtype'          => 'User',
-                            'text'              => $name,
-                            'title'             => $name,
-                            'use_notification'  => $email === '' ? false : $default_use_notif,
-                            'alternative_email' => $email,
+                            'items_id'              => $users_id_default,
+                            'itemtype'              => 'User',
+                            'text'                  => $name,
+                            'title'                 => $name,
+                            'use_notification'      => $email === '' || !$userobj->isRefusedNotificationMode(Notification_NotificationTemplate::MODE_MAIL) ? false : $default_use_notif,
```

Comment:
Not sure it should be changed here. User may want to enable notifications later and would probably expect to receive notifications for already existing tickets.

Reply:
If I understand correctly you suggest to allow the user to bypass his configuration, if needed (for a single ticket for example)

Reply:
Use case is simple: a user, that had no allowed notification type by the past, updates its preferences to allow mail notifications. This user will probably expect to receive notifications for both existing and new tickets in this case.

If you force the `false` value here, then this user will not receive notification for existing tickets unless he updates the `use_notification` value.

Also, `use_notification` is probably used for all notification types, not just `MODE_MAIL`.